src/main/java/com/example/java/maven/gameBlackJack/Player.java

```diff
@@ -0,0 +1,33 @@
+package com.example.java.maven.gameBlackJack;
+
+import java.util.ArrayList;
+
+public class Player {
```

Comment:
do you need Player and Croupier classes? Can't you use `Player` class for `Croupier` also? IMHO croupier is also a player :D

Reply:
Croupier class is no longer used